tls/s2n_quic_support.c

```diff
@@ -54,6 +54,11 @@ bool s2n_connection_is_quic_enabled(struct s2n_connection *conn)
     return (conn && conn->quic_enabled) || (conn && conn->config && conn->config->quic_enabled);
 }
 
+bool s2n_connection_is_resumption_enabled(struct s2n_connection *conn)
```

Comment:
What's your reasoning for not naming it like the existing session resumption methods? We refer to "session tickets" instead of "resumption" in both the on/off method and the callback. I'd prefer consistency.

You also call it "session tickets" in the rust bindings method, so this seems odder.